.expeditor/scripts/finish_release/create_github_release.sh

```diff
@@ -7,12 +7,15 @@ set -euo pipefail
 echo "--- :lock: Importing GPG keys"
 import_gpg_keys
 
-echo "--- :ruby: Installing 'hub' gem"
-gem install hub
+echo "--- Installing 'hub' CLI tool"
+sudo apt-get install hub
```

Comment:
Unless I'm mistaken, this isn't going to work since our `chefes/buildkite` docker image is Ubuntu 18.04 based and that would require utilizing [Snap](https://github.com/github/hub#installation)

That being the case, why not just install with our shared bash function [install_hub](https://github.com/habitat-sh/habitat/blob/1f454ed987f6be875fdbaf4b5ae048ae73af1155/.expeditor/scripts/shared.sh#L313) 😄

Reply:
```
10:06 $ docker run -it --rm --init --privileged --volume ${PWD}:${PWD} --workdir ${PWD} chefes/buildkite bash -c "source .expeditor/scripts/shared.sh && install_hub && hub --version"
--- :github: Installing hub version v2.14.1 to /bin/hub from https://github.com/github/hub/releases/download/v2.14.1/hub-linux-amd64-2.14.1.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   614    0   614    0     0   2379      0 --:--:-- --:--:-- --:--:--  2389
100 4994k  100 4994k    0     0  6002k      0 --:--:-- --:--:-- --:--:-- 16.7M
git version 2.17.1
hub version 2.14.1
✔ ~/Projects/habitat [master L|…2⚑ 12]
10:06 $
```

Reply:
Good call... I missed that earlier.